### ISSUE TITLE
"Go to Definition" does not support "local our" declarations (mod_perl)

### ISSUE THREAD

**Issue by ANSI-C** [Komodo/KomodoEdit#1929]
### Short Summary

If you try to go to the definition of a variable declared as "local our" it does not work.

### Steps to Reproduce

- declare a variable as "local our $variable"
- use it in another section of code
- select "Go to Definition" from shortcut or contextual menu

### Expected results

- it should jump to the variable definition

### Actual results

no actions

### Platform Information

*Komodo Edit or IDE?* Komodo IDE
*Komodo Version?* 10.0.1, build 89237
*Operating System (and version)?* Windows 8.1

### Additional Information
didn't test "our" or "local" alone, and "state" declarations, so don't know if they work either.

 -- eg. Error logs, screenshots, workarounds --

**Comment by ANSI-C** [comment]
This was intended for Komodo IDE, not Edit, sorry.

**Comment by ANSI-C** [comment]
Well, it seems that this is indeed the right repository.

**Comment by mitchell-as** [comment]
Yes, this is the right repository. We'll flag IDE-only issues as necessary, but this affects Komodo Edit as well. Thanks for the report.

**Comment by mitchell-as** [comment]
Note to self: After looking into this, it appears we do not handle any 'our' variable declarations other than those inside a package. It's not clear how to go about implementing this enhancement without further research.
